CONTRIBUTION.md

```diff
@@ -0,0 +1,157 @@
+[![Build Status](https://travis-ci.com/PhoenicisOrg/scripts.svg?branch=master)](https://travis-ci.com/PhoenicisOrg/scripts)
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/ff0c41daa31549e4a9bb3998ca0c87ae)](https://www.codacy.com/app/PhoenicisOrg/scripts?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=PhoenicisOrg/scripts&amp;utm_campaign=Badge_Grade)
+[![Crowdin Badge](https://d322cqt584bo4o.cloudfront.net/phoenicis-scripts/localized.svg)](https://crowdin.com/project/phoenicis-scripts)
+
+# How to add a script
+1. Fork this repository, create a new branch and clone it 
+    ```
+    git clone https://github.com/<user>/scripts.git
+    cd scripts
+    git checkout <branch>
+    ```
+
+2. Add the local checkout as a local repository in Phoenicis (see [instructions](https://phoenicisorg.github.io/phoenicis/Users/repository/#local-directory))
+
+3. Select the right category for the application
+    * Accessories
+    * Development
+    * Games
+    * Graphics
+    * Internet
+    * Multimedia
+    * Office
+    * Other
+    * Science
+  
+4. Create a new folder for your script inside that category. The folder structure should be:
+    ```
+    category
+    └── application-name
+        ├── script-name
+        │   ├── script.js
+        │   └── script.json
+        ├── miniatures
+        │   └── main.png
+        └── application.json
+    ```
+
+    Typically, `script-name` will be something like "Steam", "Online" or "v1.2".
+    
+    Even if the application name contains ®, ™ or the like, you should not use it in the folder name.
+
+5. Fill the files:
+    * [script.js](https://phoenicisorg.github.io/scripts/Develop/script-js/): actual installation script
+    * [script.json](https://phoenicisorg.github.io/scripts/Develop/script-json/): describes the installation script
+        ```json
+        {
+		"scriptName": "Online",
+		"compatibleOperatingSystems": ["MACOSX", "LINUX"],
+		"testingOperatingSystems": [],
+		"free": true,
+		"requiresPatch": false
+        }
+        ```
+    * main.png: application icon (400px x 300px)
+    * [application.json](https://phoenicisorg.github.io/scripts/Develop/application-json/): describes the application
+        ```json
+        {
+ 		"name":	"Steam",
+		"description": "Application description"
```

Comment:
It seems to me like the formatting for this line and the line before is broken

Reply:
@madoar typos will be fixed, please approach as concept -> only the design of the implementation should be discussed at the moment e.i: Using CONTRIBUTION.md is a good/bad idea, because..

Reply:
Don't worry, I think it is a good idea.